tests/unit/utils/test_json_tools.py

```diff
@@ -1,8 +1,10 @@
+from typing import Any
+
 import pytest
 
 from utils.json_tools import parse_and_check_json_markdown, parse_json_markdown
 
-correct_inputs = (
+correct_inputs: tuple[tuple[str, dict[str, Any]], ...] = (
```

Comment:
Is it necessary to have typing in tests?

Reply:
not strictly necessary

Reply:
so if you remove it, would linting fail?

Reply:
no, if we setup it correctly - can be done after we change project structure

Reply:
removed as agreed upon with @onmete